viz.yaml

```diff
@@ -176,6 +176,15 @@ process:
       col_names: wu_type_column_names
     processor: summarize_wu_data
     scripts: [scripts/process/summarize_wu_data.R]
+  -
+    id: wu_state_data_json
+    location: cache/wu_state_data.json
+    mimetype: application/json
+    drag_states: ["ID","OK","MI"]
```

Comment:
:+1: for placement of this decision in the viz.yaml. but don't we often put arguments nested within an args holder, e.g.
```
mimetype: applicaton/json
process_args:
  drag_states: ["ID","OK","MI"]
```
?
i don't know if we've actually settled on a convention for this, but i prefer having them nested so i know which info is going to the function and which is getting used by vizlab for other purposes. i realize and have no solution for the fact that things like `location` blur that line.